packages/npm-dependencies-validation/.mocharc.js

```diff
@@ -0,0 +1,10 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+
+chai.use(chaiAsPromised);
+
+module.exports = {
+  require: ["source-map-support/register"],
+  spec: "./dist/test/**/*spec.js",
+  timeout: 5000,
```

Comment:
is 5000ms timeout really needed?
are the tests slow because of npm operations?
if so even higher timeout may be needed in container/vms (github actions).

Also are there only a few tests that are very slow? or are all tests slow?
Because a per test timeout can be specified in mocha.

Reply:
It depends. On my machine sometimes some tests took 4000+ millis, but sometimes the same tests took less than 2000 millis. I think it is OK to leave 5000 millis for all tests.

Reply:
normally it is not good to have a high timeout for **all** tests.
because if there is a general issue and everything fails on timeout it can take a long while to get feedback from the tests.
Not sure if that is relevant here (what percentage of the tests are long running.